src/helperFunctions/magic.py

```diff
@@ -0,0 +1,46 @@
+"""This is a wrapper around pymagic.
+It aims to provide the same API but with the ability to load multiple magic
+files in the default api.
+"""
+from __future__ import annotations
+
+import os
+from os import PathLike
+
+import magic as pymagic
+
+from helperFunctions.fileSystem import get_src_dir
+
+# On ubuntu this is provided by the libmagic-mgc package
```

Comment:
should we try to install this package or is it always installed by default?

Reply:
No, it is a transitive dependency of `file`.